app/src/main/java/com/example/habitsmasher/ui/dashboard/EditHabitDialog.java

```diff
@@ -96,7 +96,7 @@ public void onClick(View v) {
                 // habit validator to ensure data is formatted appropriately
                 HabitValidator habitValidator = new HabitValidator(_editFragment);
 
-                if (!habitValidator.isHabitValid(habitTitle, reasonText, dateText, _tracker)) {
+                if (!habitValidator.isHabitValid(habitTitle, reasonText, dateText, _tracker, _publicPrivateButtons)) {
```

Comment:
minor: This line is getting long as well, so these parameters can also be on new lines for readability.